### DISCUSSION TITLE
How to add expiry date to a session or a record?

### DISCUSSION THREAD

**Discussion by Mr-M1M3** [pocketbase/pocketbase#779] [Q&A] [answered]
* I want to a session to be expired after a certain amount of time. Is it possible to do so?

* Is it possible to delete a record automatically after a certain amount of time?

**Comment by ganigeorgiev** [comment, marked as answer]
> I want to a session to be expired after a certain amount of time. Is it possible to do so?

PocketBase doesn't have sessions. You can change the default token duration from "Settings > Token options".


> Is it possible to delete a record automatically after a certain amount of time?

Not directly. You can do that if you use PocketBase as framework and start a simple Go timer or use a cron-like library to delete your record.
Alternatively, instead of deleting the record, you can also just restrict the access to it after some specific datetime. For example:

```
published <= '2022-10-15 00:00:00'
```
There is also a special filter field `@now` that you can use for comparison with the current datetime (eg. `published > @now`)
